secret-service/pkg/backend/secretbackend_k8s.go

```diff
@@ -295,6 +296,22 @@ func (k K8sSecretBackend) createK8sSecretObj(secret model.Secret, namespace stri
 	}
 }
 
+func (k K8sSecretBackend) GetScopes() ([]string, error) {
+	scopes, err := k.ScopesRepository.Read()
+	if err != nil {
+		return nil, err
+	}
+	scopeArray := make([]string, len(scopes.Scopes))
+
+	i := 0
+	for scope := range scopes.Scopes {
+		scopeArray[i] = scope
+		i++
+	}
```

Comment:
the for loop for copying can be simplified like so: 

```suggestion
        copy(scopeArray, scopes.Scopes)
```

Reply:
Actually `scopes.Scopes` is not `[]string`, it's ` map[string]Scope`. We have to iterate over the keys of it

Reply:
oups I thought the source was a string slice :)

Reply:
Then this one should work:
```suggestion
        scopeArray := make([]string, 0, len(scopes.Scopes)) // pre-initialize slice with capacity len(scopes.Scopes)

	for scope := range scopes.Scopes {
		scopeArray = append(scopeArray, scope)
	}
```

Reply:
done :)